Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -3769,40 +3769,15 @@ final class UsingDeclarationTests: ParserTestCase {
     )
   }
 
-  func testAccessorBlockDisambiguationMarker() {
+  func testAccessorBlockAfterPatternBindingDeclWithAttribute() {
     assertParse(
       """
-      var value = initialValue { @_accessorBlock
-        get
+      var x: Int = 2
+      {
+        @something
```

Comment:
@rintaro Do you have an idea what kind of attribute would make sense here, I can’t think of anything.

Reply:
I don't think there's valid attribute for `will/didSet` at this point, at least I couldn't find a test case in swift repo. I believe handling attributes here is mostly for recovery/diagnostics. So `@something` is fine for me, maybe an attribute with argument clause (e.g. `@available(...)`) in addition to that, for extra coverage.